app/controllers/users_controller.rb

```diff
@@ -35,10 +35,21 @@ def show
 def user_all_posts
 end
 
+def approve
```

Comment:
This is for approve the users.
Simply make the users active

Reply:
Move the logic to the model. Lines 37-39 should just be @user.approve, with approve being a method in the user.rb model file.